ci/packages/package.go

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package packages
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	log "github.com/cihub/seelog"
+	"github.com/magefile/mage/sh"
+	"github.com/mysteriumnetwork/node/ci/env"
+	"github.com/mysteriumnetwork/node/ci/storage"
+	"github.com/mysteriumnetwork/node/logconfig"
+)
+
+// PackageLinuxAmd64 builds and stores linux amd64 package
+func PackageLinuxAmd64() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	if err := packageStandalone("build/myst/myst_linux_amd64", "linux", "amd64"); err != nil {
+		return err
+	}
+	return env.IfRelease(storage.UploadArtifacts)
```

Comment:
Quite strange that packaging script start uploading by magic global constants. 
As we put all artifacts to `build/package` I would call upload tasks from CI

Reply:
Can you elaborate what do you mean by _call upload tasks from CI_?

Reply:
[Embedding logic into CI configuration scripts isn't a good idea](https://www.thoughtworks.com/radar/techniques/programming-in-your-ci-cd-tool) and that is what I was trying to fix. 
As a result, builds are fully reproducible locally with a few environment variables set.

Environment variables I think are a reasonable way to pass data into build scripts as I haven't found a better way to do that with mage. Basically there are 2 kinds of them: 
1. Build information (version, build type - PR/Tag/Snapshot) 
2. API tokens and other data used with 3rd party services (Github, S3)

Do you see any problems with this approach?

Reply:
Sorry, I just realised it's upload to S3, not a conditional artifact releasing